lighthouse-core/report/html/report-styles.css

```diff
@@ -705,11 +703,6 @@
   color: var(--color-fail-secondary);
 }
 
-/* Hide icon if there was an error */
```

Comment:
old selves seemed fairly convinced we should intentionally, but why 🤔

Reply:
https://github.com/GoogleChrome/lighthouse/pull/5077 seems to be the last time it was touched, so it's fair to say a lot has changed since then :)

Reply:
> seems to be the last time it was touched, so it's fair to say a lot has changed since then :)

It's a shame we didn't preserve much context. You were very excited @patrickhulce :)

One issue is conflation of lighthouse errors with audit failures, which is something some people already get confused about (thinking "Errror!" means their page had an error). I don't know what a better icon would be, though.

Reply:
lol, yeah it's hard to tell which subchange specifically I was most excited about there, but I remember being super jazzed about the error tooltip and no more informative scores that people have no idea what they mean, so I'm guessing it's somewhat tangential to our change here?

agreed more context would've been helpful now. I also seems worth it to try to move these two states to be *less* similar in the UI given how frequently people confuse the `Error` for their page doing something wrong.

Reply:
black hexagon (stop sign)?

Reply:
@exterkamp 	"upside down red triangle. or an X"